tests/test_model.py

```diff
@@ -116,3 +116,37 @@ def test_input_validity(self, model):
         # Check that passing a list of n_obs raises an error
         with pytest.raises(Model.InvalidParameterError):
             model.fit(t_obs, n_list, pop)
+
+    ## Post-regression
+    # ci_bootstrap
+    def test_ci_without_model_init(self, model):
+        """ Tests that an error is raised if the ci routines
+        are from an model instance that doesn't have initialized
+        it parameters or initial conditions"""
+
+        # model.is_fitted = True
```

Comment:
Should we remove this comment?

Reply:
Thanks! will implement the updates today